contact.html

```diff
@@ -1,22 +1,52 @@
 <!DOCTYPE html>
 <html lang = "en">
+	<!-- http://www.w3schools.com/html/html_layout.asp -->
+	
   <title>
   Poodle Security Training School | Contact Us
   </title>
-<body style="font-family:verdana;">
-<!-- http://www.w3schools.com/html/html_layout.asp -->
-<center>
-	<header><h1>Poodle Security Training School</h1></header>
+  <link rel="shortcut icon" href="http://www.hddevelop.com/images/pawprint.ico">
+  <meta charset="utf-8">
+	
+	
+	<style>
+	body{
+	  font-family: Verdana, sans-serif;
+	}
+	
+	.center{
+	  text-align: center;
+	}
+	
+	.bars{
+	  background-color: gray;
+	  color:white;
+	  padding: 25px 0px 25px 0px;
+	}
+	  
+	.content{
+	  padding: 25px 0px;
+	}
+	
+	p{
+	  padding: 0px 25px;
+	}
+  </style>
+	
+	
+<body>
+	<header class="center bars"><h1>Poodle Security Training School</h1></header>
```

Comment:
before, this line had no CSS and used ``<center>`` tags. apparently using that is no longer supported, and you're supposed to use CSS.

so, that's what I did. I used two classes that I created earlier, and assigned them to this ``<header>`` tag.